scene/3d/spring_arm_3d.cpp

```diff
@@ -187,6 +192,8 @@ void SpringArm3D::process_spring() {
 
 		get_world_3d()->get_direct_space_state()->cast_motion(shape_params, motion_delta, motion_delta_unsafe);
 	}
+	
```

Comment:
This line has whitespace, you need to remove it

Reply:
Sorry! Should be good now^^